querqy-core/src/main/java/querqy/rewrite/contrib/replace/ReplaceInstruction.java

```diff
@@ -42,16 +43,37 @@ public void apply(final List<CharSequence> seq, final int start, final int exclu
      * @param start            Startposition of the term in the list
      * @param exclusiveOffset  Endposition of the term in the list without offset.
      * @param replacementTerms Terms that should be used as replacement
-     * @param appliedRules     Debug information about replaced terms and their replacement
+     * @param actionLoggings   Debug information about replaced terms and their replacement
```

Comment:
matchType is missing in Javadoc `@param` annotations - the javadoc generator will not accept this. Please fix this to make the release a bit easier ;-)